src/middlewares/Validate.js

```diff
@@ -143,6 +143,30 @@ class Validate {
     ];
   }
 
+  /**
+    * Validate input
+    * @static
+    * @returns {object} errors
+    */
+  static accommodationLikesDislikes() {
+    return [
+      oneOf([
+        check('like').isIn(['true', 'false'])
+      ], 'like param should be either true or false')
+    ];
+  }
+
+  /**
+  * Validate input
```

Comment:
@victorkarangwa change the jsDoc description to be specific, for example `Validate accommodation likes and dislikes`